Makefile

```diff
@@ -46,7 +46,7 @@ help:
 	@echo '  GITHUB_USER         Which GitHub user account to use for cloning for Minikube? [default=anonymous]'
 	@echo '  MINIKUBE_CPUS       How many CPUs to allocate for Minikube? [default=2]'
 	@echo '  MINIKUBE_DISKSIZE   How much disk size to allocate for Minikube? [default=40g]'
-	@echo '  MINIKUBE_DRIVER     Which vm driver to use for Minikube? [default=kvm2]'
```

Comment:
Please amend also one documentation occurrence in `cli.py` where we recommend kvm2.